examples/weather/graphcast/train_graphcast.py

```diff
@@ -45,15 +45,16 @@
 from modulus.datapipes.climate import ERA5HDF5Datapipe
 from modulus.distributed import DistributedManager
 
-try:
-    import apex
-except:
-    pass
-
-
 # Instantiate constants, and save to JSON file
 C = Constants()
 
+# Optionally import apex
+if C.use_apex:
+    try:
+        import apex
+    except:
+        pass
```

Comment:
So now that there is a dedicated `use_apex` option, would not user want to know that apex will not be used despite this option being "on"? Maybe add a warning message or don't silence the exception?

Reply:
Thanks, added an import error.